src/mumble/CoreAudio.mm

```diff
@@ -153,9 +481,10 @@
 	s.qsCoreAudioInput = choice.toString();
 }
 
-bool CoreAudioInputRegistrar::canEcho(const QString &outputsys) const {
-	Q_UNUSED(outputsys);
-
+bool CoreAudioInputRegistrar::canEcho(int echoOption, const QString &) const {
+	if (@available(macOS 10.14, *)) {
+		if (echoOption == ECHO_CANCEL_APPLE || echoOption == ECHO_CANCEL_DEFAULT) return true;
```

Comment:
This system also returns `false` if echo cancellation is disabled even though in theory it does support echo cancellation.

Reply:
I will make ECHO_CANCEL_DISABLED being intercepted before passing into this function.